src/networks.js

```diff
@@ -0,0 +1,55 @@
+import env from './environment'
+
+const DEFAULT_IPFS_GATEWAY = 'https://ipfs.eth.aragon.network/ipfs'
+const DEFAULT_ORG_LOCATION = 'agreement3.aragonid.eth'
+
+const NETWORK_ENVIRONMENTS = new Map([
+  [
+    'ethereum',
+    {
+      chainId: 1,
+      legacyNetworkType: 'main',
+      ensAddress: '',
```

Comment:
No need to add this; Connect already has this handled for us!

Reply:
Ah ok, I was told me might need to update it in certain cases!